src/python/py/models/builder.py

```diff
@@ -186,11 +224,6 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
                 self.attention_attrs["use_rotemb_in_attn"] = True
                 self.input_names.remove("position_ids")
 
-        if self.ep in {"web"}:
```

Comment:
The defaults of `use_packed_matmul = False` and `op_type = "MultiHeadAttention"` are only changed if the specified precision and execution provider is a valid GQA configuration. Since `web` is not used in any valid GQA configuration, the defaults will remain the same.